main.nf

```diff
@@ -929,7 +956,7 @@ def get_max_cpus(requested) {
         log.warn "Maximum CPUs (${requested}) was adjusted to fit your system (${available})"
         return available
     }
-    
+
```

Comment:
You'd need to undo these stylistic changes - the hard way :)

Reply:
Oh, thats true. I forgot to remove them